src/broker/Manager.cc

```diff
@@ -31,47 +32,15 @@
 #include "zeek/telemetry/Manager.h"
 #include "zeek/util.h"
 
-#ifdef BROKER_HAS_VARIANT
-#include <broker/variant.hh>
-#endif
-
 using namespace std;
 
 namespace {
 
-broker::data&& convert_if_broker_variant(broker::data&& arg) { return std::move(arg); }
-
-broker::data& convert_if_broker_variant(broker::data& arg) { return arg; }
-
-broker::data&& convert_if_broker_variant_or_move(broker::data& arg) { return std::move(arg); }
-
-broker::vector& broker_vector_from(broker::data& arg) { return broker::get<broker::vector>(arg); }
-
-#ifdef BROKER_HAS_VARIANT
-
-broker::data convert_if_broker_variant(const broker::variant& arg) { return arg.to_data(); }
-
-broker::data convert_if_broker_variant_or_move(const broker::variant& arg) { return arg.to_data(); }
-
 broker::vector broker_vector_from(const broker::variant& arg) {
     auto tmp = arg.to_data();
     return std::move(broker::get<broker::vector>(tmp));
 }
 
-#endif
-
-// Converts a string_view into a string to make sure that we can safely call `.c_str()` on the result.
-template<class View>
-std::enable_if_t<std::is_same_v<std::decay_t<View>, std::string_view>, std::string> c_str_safe(View&& arg) {
-    return std::string{arg};
-}
```

Comment:
Is the `c_str_safe()` cleanup related to `broker::data` removal? Seems a separate commit  would make review a bit easier and give a better idea of the individual changes in the history.

Up for splitting up the existing single commit?

Reply:
> Is the c_str_safe() cleanup related to broker::data removal?

Yes, a `broker::data` contains actual `std::string` objects while a `broker::variant` only grants access to characters via `std::string_view`. Hence, we need to copy things into temporary `std::string` objects when using the Zeek formatting functions.

Reply:
> Up for splitting up the existing single commit?

Done.